release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -427,14 +427,13 @@ Cluster Monitoring Operator] for capacity planning details.
 [[ocp-311-prometheus]]
 ==== Prometheus Cluster Monitoring
 
-Prometheus cluster monitoring is now fully supported in {product-title}.
+Prometheus cluster monitoring is now fully supported in {product-title} and deployed by default into an {product-title} cluster.
 
-You can deploy Prometheus on an {product-title} cluster, collect Kubernetes and
-infrastructure metrics, and get alerts. You can see and query metrics and alerts
-on the Prometheus web dashboard.
+* Query and plot cluster metrics collected by Prometheus.
+* Receive notifications from pre-packaged alerts, enabling owners to take correctiveactions and start troubleshooting problems.
+* View pre-packaged Grafana dashboards for etcd, cluster state, and many other aspectsof cluster health.
 
-See xref:../install_config/cluster_metrics.adoc#openshift-prometheus[Prometheus
-on OpenShift] for more information.
+See xref:../install_config/prometheus_cluster_monitoring.adoc[Configuring Prometheus Cluster Monitoring] for more information.
```

Comment:
This file does not exist, which is causing the build to fail

Reply:
Yeah, it's not yet merged and currently in review. What are we doing in this case?

Reply:
@rh-max @vikram-redhat Any update there? Thanks!

Reply:
@sichvoge - The link is missing the anchor to point to. Besides that, we will need to wait to merge this till @rh-max is ready to merge his.

Reply:
@rh-max What anchor do you want me to add?

Reply:
@ahardin-rh @sichvoge Use `xref::../install_config/prometheus_cluster_monitoring.adoc#prometheus-cluster-monitoring` please.

Reply:
ok added.